src/zendesk.ts

```diff
@@ -526,6 +526,10 @@ namespace Zendesk {
     intervals?: Array<{ start_time?: number; end_time?: number }>
   }
 
+  export interface Delete {
```

Comment:
Can you explain a bit why this is needed? It doesn't make obvious sense to me.

Reply:
Whenever we kick off a new app test from a developers PR, we need to upload a new app name of the affected app, i.e. "Batch Manager <sha>" so we don't override the exiting app. This will prevent multiple app changes from overriding each other and tests potentially failing. Once the test has completed, we need to delete the app from the dev environment so we don't have a ton of existing test apps sitting around taking up space. This will allow us to scale better as we add more developers.

Reply:
We're also going to use that app name in the automation itself to know to click on that app rather than the originally named app.

Reply:
Sorry but I'm still not understanding this at all. Can we Zoom quick sometime to walk me through it?

Reply:
Yep yep. Absolutely.